### DISCUSSION TITLE
[react-aria-components] - Is is possible to show a loading state in tables?

### DISCUSSION THREAD

**Discussion by ValenCassa** [adobe/react-spectrum#5557] [Q&A]
Hi all! What the title says. I'm just trying to show a spinner when paginating, chaging filter, etc. but I can't seem a way to do it. Does anyone know if this is possible? The most obvious choice is to render it as an empty state but I don't want that since I want to keep the previous height.

**Comment by snowystinger** [comment]
You could measure the height and set it explicitly during the transition
